### PR TITLE
Fix the output of model_folder in keras: Serialisation and saving guide

### PR DESCRIPTION
**Pull request by kurianbenoy**
In the section of What the SavedModel contains:

https://keras.io/guides/serialization_and_saving/#savedmodel-format
```
[34massets[m[m         saved_model.pb [34mvariables[m[m
```
output of saved model is display not properly. It's ideal to fix this issue manually with change in markdown as the PR. So for better readability of people reading guides.

What do you think @fchollet ?

### CODE REVIEW

**Comment by kurianbenoy** [comment]
It seems like my PR is conflicting with previously merged commit. If needed, I will create a new branch and update the PR

File: examples/nlp/bidirectional_lstm_imdb.py [hunk 1]
```diff
@@ -41,12 +41,14 @@
 )
 print(len(x_train), "Training sequences")
 print(len(x_val), "Validation sequences")
+# Use pad_sequence to standardize sequence length:
```
Comment (fchollet) [inline examples/nlp/bidirectional_lstm_imdb.py:44]: This file should not be included in this PR

Comment (kurianbenoy) [reply examples/nlp/bidirectional_lstm_imdb.py:44]: ok

**Comment by kurianbenoy** [comment]
Created new PR #195
